packages/api/src/utils/github/buildProjectsQuery.ts

```diff
@@ -0,0 +1,56 @@
+import { Project } from '../../entities/Project';
+import { env } from '../../env';
+import logger from '../../logger';
+
+interface Repository {
+  id: string;
+  nameWithOwner: string;
+}
+
+interface RepositoryResponse {
+  data: {
+    nodes: Repository[];
+  };
+}
+
+export const buildProjectsQuery = async (projects: Project[]) => {
+  const repoNodeIds: String[] = [];
```

Comment:
This is a [weird one](https://stackoverflow.com/a/14727461/3739332)...

```suggestion
  const repoNodeIds: string[] = [];
```